WordPress/Classes/Utility/NotificationsManager.m

```diff
@@ -209,6 +208,17 @@ + (void)handleNotificationForApplicationLaunch:(NSDictionary *)launchOptions
         DDLogVerbose(@"Launched with a remote notification as parameter:  %@", remoteNotif);
         [[WPTabBarController sharedInstance] showNotificationsTab];
     }
+    
```

Comment:
If it is a remote notification that is being handled, the way this is structured the code for the remote notification will still be evaluated. Likely this is harmless but an optimization would be to return from inside the `if (remoteNotif)` conditional and avoid evaluating `localNotif`